src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    const stringA = String(a);
+    const stringB = String(b);
+
+    if (stringA < stringB) {
+      return -1;
+    }
+
+    if (stringA > stringB) {
+      return 1;
+    }
```

Comment:
code will be easier to read if you write default compare function separately and use it here